core/src/main/java/nu/yona/server/messaging/service/BuddyMessageDto.java

```diff
@@ -41,6 +43,17 @@ public String getMessage()
 	public static abstract class Manager extends MessageDto.Manager
 	{
 		@Override
+		protected Optional<UUID> getSenderUserAnonymizedId(UserDto actingUser, Message messageEntity)
+		{
+			BuddyMessage buddyMessageEntity = (BuddyMessage) messageEntity;
+			if (actingUser.getId().equals(buddyMessageEntity.getSenderUserId()))
```

Comment:
Because of a change in the use of {{relatedUserAnonymizedId}} in {{Message}}. It now also has this purpose: "If this message is a sent item, it is the target user of buddy copy of this message". The "related" aspect used to be interpreted as "sender", but that's not necessarily the case; it can also be the receiver.